tests/acceptance/helpscount-beacon-test.js

```
@@ -0,0 +1,49 @@
+    await animationsSettled();
+
+    // await waitUntil(() => window.Beacon, { timeout: 1000 });
+
+    assert.ok(window.Beacon, 'Helpscout Beacon is initialized');
+    assert.strictEqual(typeof window.Beacon, 'function', 'Beacon function is available');
+    window.Beacon('init', 'bb089ae9-a4ae-4114-8f7a-b660f6310158');
+    window.Beacon('info');
+    console.log('window.Beacon', window.Beacon);
+    console.log(window.Beacon('info'));
+    // await pauseTest();
+});
```

ESLint

[error] Insert `··`

(prettier/prettier)